### DISCUSSION TITLE
Can we reduce the need to write 256k1?

### DISCUSSION THREAD

**Discussion by tcharding** [rust-bitcoin/rust-bitcoin#984] [Refactoring]
As we move towards stabilizing the rust-bitcoin stack would it be useful to reduce the number of times users (and us) have to write the rather annoying identifier 'secp256k1'? Should we re-export the `secp256k1` crate in `rust-bitcoin` as `secp` so from miniscript and up we can just write `secp::PrivateKey`? Would this result in any loss of clarity?

**Comment by apoelstra** [comment]
I like it, though we should continue to re-export the full name as well.

**Reply by tcharding** [reply]
Yep definitely, cheers.

**Comment by tcharding** [comment]
Added to my todo list, will do after edition bump because edition 2018 changes the way crate exports are done.
